repos/fdbt-site/src/utils/apiUtils/userData.ts

```diff
@@ -260,6 +264,11 @@ export const getSingleTicketJson = (req: NextApiRequestWithSession, res: NextApi
         ...service,
         type: 'single',
         fareZones: getFareZones(userFareStages, matchingFareZones),
+        unassignedStops: {
+            nonReturnUnassignedStops,
+            outboundUnassignedStops: [],
```

Comment:
Do these need to be empty arrays or could they just be unset for this ticket type as they are not relevant?